### PR TITLE
Course Page [React]

### PR DESCRIPTION
**Pull request by lakshaycode22**
# Description

## Type of change

- [x] User Interface
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have ```npm run format``` before committing
- [ ] I replicated the UI with the Design 

## Screenshots (if applicable) 
Screenshot 2021-10-20 at 6 55 25 PM

Please review pull request

I am having a little trouble regarding the positioning and size of image. Please help me with that.

I also require some fonts, icons, and design elements so that the page looks exactly like the given design. 

The functionality works perfectly fine. I have currently created the course(data) array in the jsx file itself. I can put it in a separate file if you want.

ROUTE: http://localhost:3000/courses

🙂🙂

### CODE REVIEW

**Comment by shreyanshmalvya** [comment]
hey @lakshaycode22 you have also not used the SVG blog for the background and the lines that border the section, also your overall alignment of components must be the same as that of hero(home page) to maintain consistency. If you need any resources or help. please let us know!

**Comment by harkiratsm** [comment]
Please resolve the conflicts 😄

**Comment by lakshaycode22** [comment]
@harkiratsm please review 

@shreyanshmalvya for some reason the cropped blob was not visible as a vector so I converted it into a png file to use.

🙂🙂🙂

**Comment by harkiratsm** [comment]
Please resolve the conflicts 😄 @lakshaycode22

**Comment by lakshaycode22** [comment]
@harkiratsm please be specific, I think i resolved all conflicts

**Comment by shreyanshmalvya** [comment]
@lakshaycode22 or @harkiratsm can any one of you upload ScreenShots of the final build, just for confirmation?

**Comment by lakshaycode22** [comment]
> @lakshaycode22 or @harkiratsm can any one of you upload ScreenShots of the final build, just for confirmation?

Screenshot 2021-11-03 at 11 56 16 AM

**Comment by harkiratsm** [comment]
> l conflicts

image

**Comment by lakshaycode22** [comment]
@harkiratsm in the remaining conflicting files, I have just added my component to the home page.

**Comment by harkiratsm** [comment]
Hey bro ! Resolve the conflicting file If you are not able to do that then just drop a dm in discord i would have a gmeet with you 😄

**Comment by harkiratsm** [comment]
@lakshaycode22 any updates 😄 !

**Comment by lakshaycode22** [comment]
@harkiratsm hey!
i am really sorry, i didnt get the email of your comment 5 days ago and I just opened my github

yeah I need some help, I will dm you on discord

**Comment by harkiratsm** [comment]
Please remove the warning 
image

File: src/views/pages/HomeView/HomeView.js [hunk 1]
```diff
@@ -1,12 +1,13 @@
 // Landing page goes here ...
 import React from 'react';
-import { Navbar, AboutUs, Testimonials, Footer, LandingPage } from './index';
+import { Navbar, Courses, AboutUs, Testimonials, Footer } from './index';
```
Comment (harkiratsm) [inline src/views/pages/HomeView/HomeView.js:4]: Just add the below line 
```javascript
import LandingPage from './Landingpage/Hero'
```

**Comment by lakshaycode22** [comment]
i think i need to make a new branch. i will make the changes in few hours

**Review by harkiratsm — APPROVED** [review]

**Comment by harkiratsm** [comment]
LGTM 🚀

**Pull request merged**